### ISSUE TITLE
Load failed: Error: Missing required element: <scripts>!

### ISSUE THREAD

**Issue by nomisge** [jmoenig/Snap#2304]
One of my students contacted me today telling me, she couldn't open her project any more.
We were able to recover an earlier version, but with lot's of code lost.
I published the broken project as followed:
https://snap.berkeley.edu/snapsource/snap.html#present:Username=meister%20yoda&ProjectName=01_Alonzo-Standard

Is there a way of getting an xml file from projects saved in the cloud without opening them? I would be able to debug the issue.

Thanks,
Simon

**Comment by bromagosa** [comment]
By the way, here's the fixed project. I don't know what caused this, but `Sprite(3)`, who inherits from `Sprite(2)`, was missing its `<scripts>` tag. @jmoenig any idea why this could have happened?

Regardless of what caused this, a missing `<scripts>` tag could just be treated as an empty `<scripts></scripts>` one, right? We don't need to prevent the project from opening.
[01_Alonzo-Standard-fixed.zip](https://github.com/jmoenig/Snap/files/2744830/01_Alonzo-Standard-fixed.zip)

**Comment by cycomachead** [comment]
> Regardless of what caused this, a missing `<scripts>` tag could just be treated as an empty `<scripts></scripts>` one, right? We don't need to prevent the project from opening.

Agreed. Perhaps a warning? if there really was an export error, the user should at least know what sprite to inspect to see if code was missing.
